projects/demo/src/pages/kit/number/number-mask-doc.template.html

```diff
@@ -118,6 +118,28 @@
             </ng-template>
             <number-mask-doc-example-5></number-mask-doc-example-5>
         </tui-doc-example>
+
+        <tui-doc-example
+            id="minus-sign"
+            heading="Minus sign"
```

Comment:
Could you, please, move this example before `Dynamic decimal zero padding`.

It is more important and basic, and it should be placed before it.